src/cli/cli.cpp

```diff
@@ -2969,6 +2969,7 @@ otError Interpreter::ProcessPrefixAdd(uint8_t aArgsLength, char *aArgs[])
 
         SuccessOrExit(error = ParseUnsignedLong(prefixLengthStr, length));
         config.mPrefix.mLength = static_cast<uint8_t>(length);
+        VerifyOrExit(config.mPrefix.mLength <= OT_IP6_PREFIX_BITSIZE, error = OT_ERROR_INVALID_ARGS);
```

Comment:
Should we move this check into core so that this covers any user of OT API (and not just the CLI)?